### PR TITLE
Contact Integration details extra data

### PR DESCRIPTION
**Pull request by julienWebmecanik**
Added extraData to Contact Integration details to enhance display possibilities.

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features
| Bug fix?                               | yes
| New feature?                           | yes
| Deprecations?                          | yes
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 

#### Description:
A simple way to add extra data to the Contact's integration details.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create a contact
3. Create an eventListener (or edit one), and Subscribe to `CoreEvents::VIEW_INJECT_CUSTOM_TEMPLATE`
4. Add extra-data to the integrations
```php
$integrations[] = [
                'integration'           => 'Mautic',
                'integration_entity'    => 'Integration',
                'integration_entity_id' => '1234',
                'date_added'            => (new \DateTime())->format('Y-m-d H:i:s'),
                'last_sync_date'        => (new \DateTime())->format('Y-m-d H:i:s'),
                'extra-data'            => [
                    'This is extra data: 23%'
                ],
            ];

            $vars['integrations'] = $integrations;

            $event->setVars($vars);
```
5. Check the extra data in the Contact/Integrations View
Screenshot 2021-08-08 at 16 57 39

### CODE REVIEW

**Comment by RCheesley** [comment]
Thanks for the PR @julienWebmecanik and for the detailed instructions. If you have a plugin that folks could install to test with that might make it a bit easier for testers, is that possible?

**Comment by julienWebmecanik** [comment]
@RCheesley there isn't realy an easy test case I think. Any sync plungin works (like CRM plugins) but you have to configure it be able to synchronise contacts to check this PR. I don't have generic credentials for such a plugin that anyone could use.

File: app/bundles/LeadBundle/Views/Integration/index.html.php [hunk 1]
```diff
@@ -38,6 +38,10 @@
                     <dd><?php echo $view['date']->toText($details['date_added'], 'UTC', 'Y-m-d H:i:s', true); ?></dd>
                     <dt><?php echo $view['translator']->trans('mautic.integration.last_sync_date'); ?></dt>
                     <dd><?php echo $view['date']->toText($details['last_sync_date'], 'UTC', 'Y-m-d H:i:s', true); ?></dd>
+
+                    <?php foreach (($details['extra-data'] ?? []) as $extraData): ?>
+                        <dd><?php echo $extraData; ?></dd>
```
Comment (kuzmany) [inline app/bundles/LeadBundle/Views/Integration/index.html.php]: Should be better to use `$key => $value` to preserve `<dt>` `<dd>` structure of 

image

Comment (julienWebmecanik) [reply app/bundles/LeadBundle/Views/Integration/index.html.php]: True, I'll make the changes

**Pull request closed without merging**